src/tensorlake/documentai/_datasets.py

```diff
@@ -416,6 +417,112 @@ async def update_dataset_async(
         resp = await self._arequest("PUT", f"/datasets/{dataset.dataset_id}", json=body)
         return Dataset.model_validate(resp.json())
 
+    def get_dataset_data(
+        self,
+        dataset: Dataset,
+        cursor: Optional[str] = None,
+        direction: Optional[PaginationDirection] = None,
+        dataset_name: Optional[str] = None,
+        limit: Optional[int] = None,
+        file_name: Optional[str] = None,
```

Comment:
What is the point of having a file name here? Looks like you want to be able to list parsed data. Is this like a filter?